### DISCUSSION TITLE
Release/5.2.4

### DISCUSSION THREAD

**Discussion by Athanasius** [EDCD/EDMarketConnector#1373] [Announcement]
* We now test against, and package with, Python 3.9.9.

  **As a consequence of this we no longer support Windows 7.  
  This is due to [Python 3.9.x itself not supporting Windows 7](https://www.python.org/downloads/windows/).
  The application (both EDMarketConnector.exe and EDMC.exe) will crash on startup due to a missing DLL.**

  This should have no other impact on users or plugin developers, other than the latter now being free to use features that were introduced since the Python 3.7 series.

  Developers can check the contents of the `.python-version` file in the source (it's not distributed with the Windows installer) for the
  currently used version in a given branch.

---

Release 5.2.4
===
This is a *very* minor update that simply imports the latest versions of data files so that some niche functionality works properly.

* Update `commodity.csv` and `rare_commodity.csv` from the latest [EDCD/FDevIDs](https://github.com/EDCD/FDevIDs) versions.  This addresses an issue with export of market data in Trade Dangerous format containing `OnionHeadC` rather than the correct name, `Onionhead Gamma Strain`, that Trade Dangerous is expecting.

  This will only have affected Trade Dangerous users who use EDMarketConnector as a source of market data.

<hr /><em>This discussion was created from the release <a href='https://github.com/EDCD/EDMarketConnector/releases/tag/Release/5.2.4'>Release/5.2.4</a>.</em>

**Comment by gittwitt** [comment]
Upgraded to 5.2.4 this evening and Norton 360 detected the following:
![1](https://user-images.githubusercontent.com/96503209/147010177-facc8e9e-c790-447d-b4ec-c84f840a4849.PNG)
![4](https://user-images.githubusercontent.com/96503209/147010178-3549d16d-8ee6-4064-bcf8-f3d08f1b676b.PNG)
![3](https://user-images.githubusercontent.com/96503209/147010180-bcc9d448-f8d3-4ef3-9adb-34996057a20e.PNG)
![2](https://user-images.githubusercontent.com/96503209/147010181-6099c8cd-944c-40dd-8009-5ea09c18d550.PNG)
Hopefully a false positive? Submitting this to you for further investigation.

**Reply by Athanasius** [reply]
As per https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting#installer-and-or-executables-flagged-as-malicious-viruses , yes, it will be a false positive.  Compared to 5.2.3 there were zero code changes, as it only updated two data files.  It would take a supply-chain attack against GitHub/PyPi to cause this.

**Comment by ghost** [comment]
Regarding Python 3.9, Windows 7 users may try this DLL: https://github.com/nalexandru/api-ms-win-core-path-HACK put it in the same directory as the main exe. If that is the only missing DLL then it will work just fine.
